### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by JSCU-CNI** [fox-it/dissect.target#516]
As of #495 the following 20 tests seem to fail for the `main` branch. This seems to happen on clean pypy 3.9 and cpython 3.11 tox installs.

```
FAILED tests/helpers/test_docs.py::test_docs_plugin_functions_desc - AssertionError: assert 3 == 2
FAILED tests/plugins/apps/webserver/test_iis.py::test_iis_plugin_iis_format - assert 0 == 10
FAILED tests/plugins/apps/webserver/test_iis.py::test_plugins_apps_webservers_iis_access_iis_format - assert 0 == 10
FAILED tests/plugins/apps/webserver/test_webserver.py::test_access_logs_webserver_namespace_windows - assert 0 == 10
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[-None-] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[/systemroot/test-None-/%systemroot%/test] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[\\systemroot\\test-None-/%systemroot%/test] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[/??/C:/test-None-C:/test] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[\\??\\C:\\test-None-C:/test] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows["\\??\\C:\\windows\\system32\\calc.exe" -args-None-C:/windows/system32/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[\\\\?\\C:\\Users\\Testpath\\Testfile-None-C:/Users/Testpath/Testfile] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[C:/windows/system32/calc.exe -args-None-C:/windows/system32/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[C:/some dir with spaces/calc.exe-None-C:/some dir with spaces/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[C:/some dir with spaces/calc.exe -with -arguments-None-C:/some dir with spaces/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[C:/windows/system32/calc -args-None-C:/windows/system32/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[calc -args-None-sysvol/windows/syswow64/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[calc-no_env_use_fallback-sysvol/windows/system32/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[calc-1337-sysvol/some_dir/calc.exe] - RuntimeError: generator raised StopIteration
FAILED tests/plugins/filesystem/test_resolver.py::test_resolver_plugin_resolve_windows[foo-1337-sysvol/other_dir/foo.bat] - RuntimeError: generator raised StopIteration
FAILED tests/test_registration.py::test_registration[loader.py-plugin_list0-TestLoader] - AssertionError: assert 'TestLoader' == 'MultiRawLoader'

**Comment by Schamper** [comment]
We've noticed this and it appears to be because `pytest` has been updated to 8.0.0. Initial investigation suggests that some of our tests rely or incorrectly handle side-effects from other tests, and `pytest` is now more punishing :smile: we're working on fixing our tests!

**Comment by JSCU-CNI** [comment]
Thanks for the update! Pinning `pytest` to `7.4.4` indeed seems to fix this.

**Comment by JSCU-CNI** [comment]
Fixed in #517.
